Editor/AGS.Editor/Components/GuiComponent.cs

```diff
@@ -209,6 +209,19 @@ public override void ShowItemPaneByName(string name)
                     ShowOrAddPane(g);
                     return;
                 }
+                
+                foreach(GUIControl gctrl in g.Controls)
+                {
+                    if(gctrl.Name == name)
+                    {
+                        _guiController.ProjectTree.SelectNode(this, GetNodeID(g));
+                        ShowOrAddPane(g);
+                        GUIEditor guiEditor = _documents[g]?.Control as GUIEditor;
+                        guiEditor?.SelectGUIControl(gctrl);

```

Comment:
I think that calling SelectGUIControl may be redundant, if i remember right the gui editor should react to property grid selecting a new object.

Reply:
You are correct, adjusted it!